### PR TITLE
Build 2024-07-03 main

### PR DESCRIPTION
**Pull request by twhetzel**
Resolves #ISSUE(s).

## Overview

Build data from `main` for next week's Alignment Build Week for the Mondo release cycle.
(built as: sh run.sh make build-mondo-ingest -B > output_2024-07-03.log 2>&1)

### Documentation
Was the documentation added/updated under `docs/`?
- [ ] Yes
- [x] No, updates to the docs were not necessary after careful consideration

### QC
Was the full pipeline run before submitting this PR using `sh run.sh make build-mondo-ingest` on this branch (after 
`docker pull obolibrary/odkfull:dev`), and no errors occurred?
- [x] Yes
- [ ] No, there are no functional (code-related) changes to the pipeline in the PR, so no re-run is necessary
   
### New Packages
Were any new *Python packages* added?
- [ ] Yes, [requirements.txt.full](https://github.com/INCATools/ontology-development-kit/blob/master/requirements.txt.full) was updated and an [ODK issue](https://github.com/INCATools/ontology-development-kit/issues/new) submitted
- [x] No

Were any other *non-Python packages* added?
- [ ] Yes, [Dockerfile](https://github.com/INCATools/ontology-development-kit/blob/master/Dockerfile) updated and an [ODK issue](https://github.com/INCATools/ontology-development-kit/issues/new) submitted
- [x] No

### PR Review and Conversations Resolved
Has the PR been sufficiently reviewed by at least 1 team member of the Mondo Technical team and all threads resolved?
- [ ] Yes

### CODE REVIEW

File: src/ontology/slurp/ordo.tsv
Comment (matentzn) [inline src/ontology/slurp/ordo.tsv:1]: Not toooo bad! Just 102! :D Not as terrible as in other years where we had 250 or more.

**Review by matentzn — APPROVED** [review]
Some comments for clarification but looks great!

File: src/ontology/external/nord.robot.tsv
[the remark this answers is not in the document]
Comment (joeflack4) [reply src/ontology/external/nord.robot.tsv:1]: Do we need to add some sorting to this then, I suppose?

**Review by joeflack4 — APPROVED** [review]
LGTM!

File: src/ontology/lexmatch/split-mapping-set/mondo_broadmatch_doid.tsv
Comment (joeflack4) [inline src/ontology/lexmatch/split-mapping-set/mondo_broadmatch_doid.tsv:1]: Deleted file, but I'm guessing there just are no such current cases.

**Pull request merged**